erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -499,19 +499,22 @@ def make_rm_stock_entry(purchase_order, rm_items):
 			for rm_item_data in rm_items_list:
 				if rm_item_data["item_code"] == item_code:
 					rm_item_code = rm_item_data["rm_item_code"]
-					items_dict = {
-						rm_item_code: {
-							"po_detail": rm_item_data.get("name"),
-							"item_name": rm_item_data["item_name"],
-							"description": item_wh.get(rm_item_code, {}).get('description', ""),
-							'qty': rm_item_data["qty"],
-							'from_warehouse': rm_item_data["warehouse"],
-							'stock_uom': rm_item_data["stock_uom"],
-							'main_item_code': rm_item_data["item_code"],
-							'allow_alternative_item': item_wh.get(rm_item_code, {}).get('allow_alternative_item')
-						}
-					}
-					stock_entry.add_to_stock_entry_detail(items_dict)
+					args = frappe._dict({
+						"po_detail": rm_item_data.get("name"),
+						"item_code": rm_item_code,
+						"item_name": rm_item_data["item_name"],
+						"description": item_wh.get(rm_item_code, {}).get('description', ""),
+						'qty': rm_item_data["qty"],
+						's_warehouse': rm_item_data["warehouse"],
+						't_warehouse': purchase_order.supplier_warehouse,
+						'stock_uom': rm_item_data["stock_uom"],
+						'uom': rm_item_data["stock_uom"],
+						'subcontracted_item': rm_item_data["item_code"],
+						'allow_alternative_item': item_wh.get(rm_item_code, {}).get('allow_alternative_item')
```

Comment:
Sorry 🙈 , quote consistency.
```suggestion
						"qty": rm_item_data["qty"],
						"s_warehouse": rm_item_data["warehouse"],
						"t_warehouse": purchase_order.supplier_warehouse,
						"stock_uom": rm_item_data["stock_uom"],
						"uom": rm_item_data["stock_uom"],
						"subcontracted_item": rm_item_data["item_code"],
						"allow_alternative_item": item_wh.get(rm_item_code, {}).get("allow_alternative_item")
```